app/src/main/java/com/example/stretchy/features/executetraining/sound/data/SoundType.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.stretchy.features.executetraining.sound.data
+
+import com.example.stretchy.common.OneTimeEvent
+
+sealed class SoundType(override var value: Unit? = Unit) : OneTimeEvent<Unit>() {
+    class TrainingCompletedEvent : SoundType()
+
+    class ActivityFinishesEvent : SoundType()
```

Comment:
I am also unsure about `ActivityFinishes`. Can the name be more precise in time? `Finishes` is when? When it finished? or before finishing (so when), or is "start finishing"? I am not sure